src/service/profile.js

```diff
@@ -151,6 +153,7 @@ export const setFollow = async (data) => {
         resolve(res.data);
       })
       .catch((err) => {
+        console.error(err.response);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, make sure to check if `err.response` exists before logging it.
```diff
- console.error(err.response);
+ if (err.response) {
+   console.error(err.response);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->